committee_admissions/admissions/tests/test_api.py

```diff
@@ -71,7 +85,7 @@ def test_pleb_cannot_edit_group(self):
         self.client.force_authenticate(user=self.pleb)
 
         res = self.client.patch(
-            reverse("group-detail", kwargs={"pk": self.arrkom.pk}),
+            self.urls["group-detail"].format(self.arrkom.pk),
```

Comment:
You really _should_ use the `reverse` function here and not create the url yourself, but it's not the end of the world.

Reply:
Had some problems getting that to work when I used [this approach](https://github.com/webkom/committee-admissions/blob/multiple-admissions/committee_admissions/urls.py#L31) to define the application URL, but I'll take a crack at getting it to work on todays workshop.   
And I'll fix it for the group-detail as that route should still work with the old usage of the reverse method.